go/bind/keybase.go

```diff
@@ -121,9 +151,9 @@ func setInited() {
 // InitOnce runs the Keybase services (only runs one time)
 func InitOnce(homeDir, mobileSharedHome, logFile, runModeStr string,
 	accessGroupOverride bool, dnsNSFetcher ExternalDNSNSFetcher, nvh NativeVideoHelper,
-	mobileOsVersion string, isIPad bool) {
+	mobileOsVersion string, isIPad bool, installReferrerListener NativeInstallReferrerListener) {
```

Comment:
i assume this doesn't break ios

Reply:
I did not check, as I cannot build for ios... I assumed CI would check for that. 
If not, can you tell me what to do here please?

Reply:
Ci doesn't build or run iOS at all. Maybe someone from your team can try it

Reply:
I'm pretty sure this _will_ break iOS, but not on this line. Instead it's in `Init`. `InitOnce` is only called on Android.

To fix iOS, you'll need to pass the new parameter to this line: https://github.com/keybase/client/blob/6b357fbc3f32d338ec7402e4c031a87461710be2/shared/ios/Keybase/Engine.m#L62

Reply:
that is already part of this pr